cards.js

```diff
@@ -59,6 +60,9 @@ class Card {
   isJoker() {
     return this.type == Card.TYPE_JOKER;
   }
+  copy(){
+    return new Card(this.type, this.suit, this.rank);
+  }
 }
 
 Card.RANK_ACE = 1;
```

Comment:
@towergame šeit jābūt 1?

Reply:
Noziedojiet, lūdzu, konstruktoru, nabaga klasei.

Reply:
Jaa. @sakars

Reply:
Fine.
I'ļl do it myself.

Reply:
> Noziedojiet, lūdzu, konstruktoru, nabaga klasei.

nepareizais thread mansk

Reply:
@towergame Vai RANK_ACE=1

Reply:
Jā, sorry par spamu nevietā.